lib/app.js

```diff
@@ -40,7 +40,13 @@ function app (opts) {
       .catch(reject);
   });
 }
-
+/*
```

Comment:
this is out of scope for this PR, but this sort of info should perhaps be in other doc, and we could eventually add some sort of script to do that work instead of relying on those websites